docs/installation/upgrading.rst

```diff
@@ -6,26 +6,80 @@ Upgrading KSQL
 Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
 capacity to take over temporarily for unavailable, restarting servers.
 
+Upgrading to ksqlDB 5.5 from KSQL 5.4
```

Comment:
Would be good to include general 5.4 ksql to 5.5 ksqldb upgrade info from https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/ - or have a link to that upgrade doc at the beginning of 5.4->5.5 section